test/blocks/gnav/mocks/breadcrumbs/from-ancestor-file/level1/level2/level3/page.test.html

```diff
@@ -0,0 +1,50 @@
+<html>
+<head>
+    <meta name="gnav-source" content="/test/blocks/gnav/mocks/gnav">
+    <link rel="stylesheet" href="/libs/blocks/gnav/gnav.css">
+</head>
+<body>
+<header class="gnav" data-block-name="gnav" data-gnav-source="/test/blocks/gnav/mocks/gnav">
+</header>
+<main>
+    <div class="section-wrapper">
+        <div>
+            <h1 id="welcome-to-the-gnav-test">Welcome to the Gnav/Breadcrumbs Test</h1>
+            <div id="wrong" class="gnav" data-block-name="gnav" data-gnav-source="/test/blocks/gnav/mocks/wrong-gnav"></div>
+            <div style="height: 1000px">
+                - no breadcrumbs block defined for this page.<br/>
+                - a breadscrumbs file is defined in one of the ancestor folders
+            </div>
+        </div>
+    </div>
+</main>
+</body>
+<script type="module">
+  import { runTests } from '@web/test-runner-mocha';
+  import { expect } from '@esm-bundle/chai';
+  import { delay } from '../../../../../../../../helpers/waitfor.js';
+  import gnav from '../../../../../../../../../libs/blocks/gnav/gnav.js';
+  import getItems from '../../../../utils.js';
+  import { setConfig } from '../../../../../../../../../libs/utils/utils.js';
+
+  runTests(() => {
+    const config = { locales: { '': { ietf: 'en-US', tk: 'hah7vzn.css' } } };
+    setConfig(config);
+
+    describe('breadcrumbs', () => {
+      it('breadcrumbs defined in the breadcrumbs file in an ancestor folder', async () => {
+        await gnav(document.querySelector('header'));
+        await delay(300);
+        const expected = [
+          {href: 'http://www.google.com/', title: 'from-ancestor-file'},
+          {href: 'http://www.adobe.com/', title: 'Actors'},
+          {href: 'http://www.day.com/', title: 'Players'},
+          {href: '', title: 'page.test.html'},
+        ];
+        expect(getItems()).to.eql(expected);
+      });
+    });
+  });
+</script>
+</html>
```

Comment:
Why are the tests in an html page?

Reply:
Each breadcrumbs test relies not just on a single page but on a  hierarchy of pages. For example the above test verifies that the breadcrumbs is correctly generated when the breadcrumbs is defined in an ancestor file. The test relies on pages at each level below  `from-ancestor-file/level1/level2/level3`. In this case having the test in the html file where the breadcrumbs is generated makes sense to me for ease of readability and maintainability.

Reply:
Most of our tests are written in js test files and they import html pages as needed. However, I do see that they've used this pattern before with caas so looks like the Milo team accepts html page tests too